.teamcity/components/generated/services.kt

```diff
@@ -92,7 +92,7 @@ var services = mapOf(
         "network" to "Network",
         "networkfunction" to "Network Function",
         "newrelic" to "New Relic",
-        "nginx" to "Nginx",
+        "nginx" to "NGINX",
```

Comment:
This is a generated file and causing the generation check to fail. Can you remove this change, it'll be updated automatically since the casing in the `registration.go` was updated.
```suggestion
```